File: Server/Core/Models/Saint.cs
namespace Core.Models;

public class Saint
{
    public int Id { get; set; }
    public required string Name { get; set; }
    public required string Country { get; set; }
    public required int Century { get; set; }
    public required string Image { get; set; }
    public required string Description { get; set; }
    public required string MarkdownPath { get; set; }
    public string? Title { get; set; }
    public DateOnly? FeastDay { get; set; }
    public string? PatronOf { get; set; }
    public int? ReligiousOrderId { get; set; }
    public ReligiousOrder? ReligiousOrder { get; set; }
    public required string Slug { get; set; }
    public List<Tag> Tags { get; set; } = [];
}

File: Client/src/app/interfaces/saint.ts
import { ReligiousOrder } from './religious-order';
import { Tag } from './tag';

export interface Saint {
  id: number;
  name: string;
  country: string;
  century: number;
  image: string;
  description: string;
  markdownPath: string;
  title?: string | null;
  feastDay?: string | null; 
  patronOf?: string | null;
  religiousOrder?: ReligiousOrder | null;
  slug: string;
  tags: Tag[];
}
